overlays/bdc-overlay.yaml

```diff
@@ -0,0 +1,13 @@
+machines:
+  '0':
+    constraints: cores=2 mem=4G root-disk=16G
+    series: bionic
+  '1':
+    constraints: cores=8 mem=64G root-disk=100G
+    series: bionic
+applications:
+  kubernetes-master:
+    options:
+      channel: 1.18/stable
```

Comment:
This seems like it could lead to the channel for the master and worker getting out of sync.

Reply:
yeah, I can remove some of the unnecessary stuff